### PR TITLE
fix: rakuten: sync property mapping sourcekeys to rudderstack standard spec

### PR DESCRIPTION
**Pull request by anantjain45823**
## What are the changes introduced in this PR?
Syncing properties mapping to standard rudderstack spec

## What is the related Linear task?

Resolves INT-1675

## Please explain the objectives of your changes below

Put down any required details on the broader aspect of your changes. If there are any dependent changes, **mandatorily** mention them here

### Any changes to existing capabilities/behaviour, mention the reason & what are the changes ?

N/A

### Any new dependencies introduced with this change?

N/A

### Any new generic utility introduced or modified. Please explain the changes.

N/A

### Any technical or performance related pointers to consider with the change?

N/A

@coderabbitai review

<hr>

### Developer checklist

- [x] My code follows the style guidelines of this project

- [x] **No breaking changes are being introduced.**

- [x] All related docs linked with the PR?

- [x] All changes manually tested?

- [x] Any documentation changes needed with this change?

- [x] Is the PR limited to 10 file changes?

- [x] Is the PR limited to one linear task?

- [x] Are relevant unit and component test-cases added?

### Reviewer checklist

- [x] Is the type of change in the PR title appropriate as per the changes?

- [x] Verified that there are no credentials or confidential data exposed with the changes.

### CODE REVIEW

**Comment by devops-github-rudderstack** [comment]
Test report for this run is available at: https://test-integrations-dev.s3.amazonaws.com/integrations-test-reports/rudder-transformer/3129/test-report.html

File: src/cdk/v2/destinations/rakuten/data/propertiesMapping.json [hunk 1]
```diff
@@ -1,41 +1,45 @@
 [
   {
-    "sourceKeys": "properties.orderId",
+    "sourceKeys": ["properties.orderId", "properties.order_id"],
```
Comment (saikumarrs) [inline src/cdk/v2/destinations/rakuten/data/propertiesMapping.json:3]: We should prefer the fields from standard spec.

Comment (saikumarrs) [reply src/cdk/v2/destinations/rakuten/data/propertiesMapping.json:3]: Same for other fields as well.

**Review by saikumarrs — CHANGES_REQUESTED** [review]

File: src/cdk/v2/destinations/rakuten/data/propertiesMapping.json (hunk 1 above)
Comment (anantjain45823) [reply src/cdk/v2/destinations/rakuten/data/propertiesMapping.json]: done

**Review by shrouti1507 — APPROVED** [review]

**Review by saikumarrs — APPROVED** [review]

**Pull request merged**